lib/assets/data_table/main.js

```diff
@@ -116,7 +123,14 @@ export function init(ctx, data) {
       orderBy(newKey) {
         this.loading = true;
         const [key, order] = reorder(this.data.content.order_by, this.data.content.order, newKey)
-        ctx.pushEvent("order_by", { key, order });
+        const label = key ? this.data.content.columns[key].label : null
+        ctx.pushEvent("order_by", { key, order, label});
+      },
+
+      orderedBy() {
```

Comment:
Let's make it clearer that it returns a human-readable string :)

```suggestion
      orderDescription() {
```